client/server/middleware/assets.js

```diff
@@ -58,10 +58,14 @@ export default () => {
 				return EMPTY_ASSETS;
 			}
 
-			const allTheFiles = chunk.files.concat(
-				flatten( chunk.siblings.map( ( sibling ) => getChunkById( assets, sibling ).files ) )
-			);
+			if ( ! chunk.chunks ) {
+				console.warn( 'cannot find the chunks of the chunk ' + chunkName );
```

Comment:
Better wording:
```
cannot find the chunks of the chunk group xxx
```
In the Calypso codebase we often don't correctly distinguish between a "chunk" and a "chunk group". For historical reasons, because we've been using webpack for a long time and "chunk groups" were added only in version 4 I think.

Reply:
Ok! Updated, thank you!